docs/language/api.rst

```diff
@@ -1210,16 +1210,81 @@ alternatively be written using the apostrophe (``'``) symbol.
 require
 -------
 
-``require`` is used to import macros from a given module. It takes at least one
-parameter specifying the module which macros should be imported. Multiple
-modules can be imported with a single ``require``.
+``require`` is used to import macros from one or more given modules. It allows
+parameters in all the same formats as ``import``. The ``require`` form itself
```

Comment:
Embarassingly, I had no idea that `from foo import bar as baz` is legal Python, let alone supported in Hy. I'll add that form for `require`.